packages/components/src/RecurringSelect/DateCellCheckbox.module.css

```
@@ -54,7 +54,7 @@
 
 .dayCheckboxInput:disabled + .dayCheckbox {
   color: var(--color-disabled);
-  background: var(--color-surface--background);
+  background: var(--color-interactive--background);
```

```suggestion
  background: var(--color-disabled--secondary);
```

Similar visual bug to the input!